plugins/modules/iam_user_info.py

```diff
@@ -103,15 +103,26 @@
             type: dict
             returned: if user exists
             sample: '{"Env": "Prod"}'
+        console_access:
+            description: If user has access to log in from AWS default console.
+            returned: always
+            type: bool
+            sample: "true"
 """
 
 from ansible_collections.amazon.aws.plugins.module_utils.iam import AnsibleIAMError
 from ansible_collections.amazon.aws.plugins.module_utils.iam import get_iam_group
 from ansible_collections.amazon.aws.plugins.module_utils.iam import get_iam_user
 from ansible_collections.amazon.aws.plugins.module_utils.iam import list_iam_users
 from ansible_collections.amazon.aws.plugins.module_utils.iam import normalize_iam_user
-from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule
+from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule, is_boto3_error_code
 
+def check_console_access(connection, user_name):
+# add function to check if a user has or not access to login via console
+    try:
+        return connection.get_login_profile(UserName=user_name)['LoginProfile']
+    except is_boto3_error_code("NoSuchEntity"):
+        return {}
```

Comment:
```suggestion
@IAMErrorHandler.list_error_handler("get login profile", {}) 
@AWSRetry.jittered_backoff()
def check_console_access(connection, user_name):
    return connection.get_login_profile(UserName=user_name)['LoginProfile']
```

Apologies for the lack of documentation around the error handler decorator, the IAM modules are the first place they've been implemented...